server/test/integration/drink_test.go

```diff
@@ -3,33 +3,46 @@ package integration
 import (
 	"net/http"
 	"net/http/httptest"
+	"os"
 	"server/graphql"
 	"server/router"
 	"testing"
 
+	"github.com/gin-gonic/gin"
 	"github.com/gin-gonic/gin/binding"
 	"github.com/stretchr/testify/assert"
 )
 
 const (
-	endpoint = "/api/drink"
+	endpointDrink = "/api/drink"
+	endpointEnjoy = "/api/enjoy"
```

Comment:
This may be a bad practice to set consts that are not related all in one. Setting  `endpointEnjoy` in `enjoy_test.go` doesn't work ?

Reply:
I fix it in another branch